modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa/ghost_spawn.dm

```diff
@@ -33,13 +33,13 @@
 	outfit = /datum/outfit/security_guard
 	you_are_text = "You are a security guard in a top secret government facility. You blacked out. Now, you have woken up to the horrors that lay within. DO NOT TRY TO EXPLORE THE LEVEL. STAY AROUND YOUR AREA."
 
-/obj/item/clothing/under/rank/security/peacekeeper/junior/sol/blackmesa
+/obj/item/clothing/under/rank/security/peacekeeper/sol/traffic/blackmesa
```

Comment:
why is this pathed like this? this is some /obj/structure/bed/chair/stool stuff

Reply:
I'm just removing the /junior, the others are all subtypes of the peacekeeper and im hesitant to touch that many itempaths in a PR thats meant to just remove redundant shit (which is, agian, the /junior)